adapters/bazaar/index.js

```diff
@@ -0,0 +1,115 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class Bazaar {
+    // stands for Ethereum name service
+    constructor() {
+        this.name = "bazaar";
+        this.symbol = "SKILL";
+        this.token = "0x154a9f9cbd3449ad22fdae23044319d6ef2a1fab";
+        this.protocol = "bsc";
+        this.block = 8114624;
+        this.contract = "0x90099dA42806b21128A094C713347C7885aF79e2";
```

Comment:
Please lowercase value of `this.contract`

Reply:
done